pass.el

```diff
@@ -379,6 +429,19 @@ If SUBDIR is nil, return the entries of `(password-store-dir)'."
     (define-key map (kbd "C-c C-w") #'pass-view-copy-password)
     map))
 
+(defun pass-view-entry-name (&optional buffer)
+  "Return the entry name for BUFFER.
+Iff BUFFER has `pass-view-mode' enable."
```

Comment:
You have a typo here

Reply:
I was going for "if and only if", as in https://en.wikipedia.org/wiki/If_and_only_if, but I'll change it to make it clearer.

Reply:
'iff' is quite fine with me, but I think 'enabled' should replace 'enable'. Let's blame @NicolasPetton who was too lazy to tell you which typo he was referring to :-).

Reply:
Fixed, I noticed the "enable" typo when I was rewriting the whole sentence. Thanks!